jobs/cam/stats/JEVS_CAM_STATS

```diff
@@ -52,6 +52,7 @@ export COMINobs=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
 export COMOUT=${COMOUT:-$(compath.py ${NET}/${evs_ver}/${STEP}/${COMPONENT}/${RUN})}
 export COMOUTsmall=${COMOUTsmall:-$(compath.py ${NET}/${evs_ver}/${STEP}/${COMPONENT}/${RUN}.${VDATE}/${MODELNAME}/${VERIF_CASE})}
 export COMOUTfinal=${COMOUTfinal:-$(compath.py ${NET}/${evs_ver}/${STEP}/${COMPONENT}/${MODELNAME}.${VDATE})}
+mkdir -p $COMOUTfinal
```

Comment:
This is a good start, but the mkdir command needs to be for $COMOUT specifically. See examples in $PACKAGEROOT for other models or networks. The reason is b/c COMOUTsmall and COMOUTfinal depend on the existence of COMOUT (COMOUT is in their parent path).  COMOUTsmall and COMOUTfinal do not need to be created at the J-job level. Downstream scripting properly creates those dirs before there are any attempts to write to them (lines 103 and 108 of exevs_nam_firewxnest_grid2obs_stats.sh).
It also does not hurt to adopt the NCO option of "-m 775". So I suggest line 55 should be:
 `mkdir -m 775 -p $COMOUT`

Reply:
I changed the mkdir line to this specification.

I don't see the pycache directory at all in the ush directory:

https://github.com/PerryShafran-NOAA/EVS/tree/feature/firewx/ush/cam

Perry